scripts/file2hex.py

```diff
@@ -44,6 +46,7 @@ def main():
         with io.BytesIO() as content:
             with open(args.file, 'rb') as fg:
                 with gzip.GzipFile(fileobj=content, mode='w',
+                                   mtime=args.gzip_mtime,
```

Comment:
I can't imagine that the embedded FW would be interested in the timestamp of the host system source file.

IMHO we should simplify by always setting mtime=0. Then the next user that wants reproducible FW won't have to discover that he needs to set --gzip-mtime=0.

Reply:
Will do unless anyone objects. In that case I will also simplify the tests/ that currently ignores the timestamp.

Reply:
AFAIK mtime=0 will still allocate space for a zero'd timestamp, so the test should/can continue to ignore the timestamp. Or it could assert it to be zero if it wanted to.

Reply:
Done and ready for re-review.